### PR TITLE
Adding some more missing renames for Transform3D and Quaternion

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Found  few more places where Transform and Quat haven't been renamed causing issues with parsing GDScript.
Bound to be more :)

### CODE REVIEW

File: modules/visual_script/visual_script_editor.cpp [hunk 1]
```diff
@@ -627,7 +627,7 @@ void VisualScriptEditor::_update_graph(int p_only_id) {
 		Control::get_theme_icon("Quaternion", "EditorIcons"),
 		Control::get_theme_icon("AABB", "EditorIcons"),
 		Control::get_theme_icon("Basis", "EditorIcons"),
-		Control::get_theme_icon("Transform", "EditorIcons"),
+		Control::get_theme_icon("Transform3D", "EditorIcons"),
 		Control::get_theme_icon("Color", "EditorIcons"),
```
Comment (BastiaanOlij) [inline modules/visual_script/visual_script_editor.cpp:631]: Not sure about this one, @vnen , @aaronfranke are these icons renamed? I noticed there are a bunch more like this in `./editor`

Comment (akien-mga) [reply modules/visual_script/visual_script_editor.cpp:631]: `Transform.svg` hasn't been renamed, would be good to fix it too in this PR:
```
git mv editor/icons/Transform.svg editor/icons/Transform3D.svg
```

Comment (akien-mga) [reply modules/visual_script/visual_script_editor.cpp:631]: Also found:
```
editor/connections_dialog.cpp
449:    type_list->add_item("Transform", Variant::TRANSFORM3D);

editor/property_selector.cpp
138:                    search_options->get_theme_icon("Transform", "EditorIcons"),
```
And plenty of `"Transform"` in `editor/plugins/visual_shader_editor_plugin.cpp` (some referring to the icon, others might be for different stuff).

**Comment by akien-mga** [comment]
Also found:
```
editor/connections_dialog.cpp
449:    type_list->add_item("Transform", Variant::TRANSFORM3D);

editor/property_selector.cpp
138:                    search_options->get_theme_icon("Transform", "EditorIcons"),
```
And plenty of "Transform" in `editor/plugins/visual_shader_editor_plugin.cpp` (some referring to the icon, others might be for different stuff).

File: modules/visual_script/visual_script_editor.cpp (hunk 1 above)
Comment (aaronfranke) [reply modules/visual_script/visual_script_editor.cpp:631]: The SVG file will need to be renamed.

**Comment by akien-mga** [comment]
Force pushed an update with more fixes which I listed above. For `editor/plugins/visual_shader_editor_plugin.cpp` I only changed the theme icon strings, I left the other "Transform" references which might refer to transforms as a noun and not as the actual type.

File: editor/connections_dialog.cpp [hunk 2]
```diff
@@ -446,7 +446,7 @@ ConnectDialog::ConnectDialog() {
 	type_list->add_item("Quaternion", Variant::QUATERNION);
 	type_list->add_item("AABB", Variant::AABB);
 	type_list->add_item("Basis", Variant::BASIS);
-	type_list->add_item("Transform", Variant::TRANSFORM3D);
+	type_list->add_item("Transform3D", Variant::TRANSFORM3D);
```
Comment (akien-mga) [inline editor/connections_dialog.cpp:449]: For the reference, this dialog doesn't include a number of Variant types: Transform2D, Vector2i, Vector3i, Rect2i, and everything after NodePath.
I don't know if this is intentional and/or if this should be fixed.

**Review by akien-mga — APPROVED** [review]

**Comment by aaronfranke** [comment]
I can confirm that the highlighting and functionality for Transform3D and Quaternion is now correct in GDScript.

**Pull request merged**